assets/src/wizards/subscriptions/style.scss

```diff
@@ -0,0 +1,134 @@
+/**
+ * Subscriptions Wizard styles.
+ */
+
+#newspack-subscriptions-wizard {
+	header.muriel-formatted-header {
+		margin-bottom: 32px;
+	}
+}
+
+/**
+ * Subscription management screen.
+ */
+.newspack-manage-subscriptions-screen {
+	max-width: 440px;
+	margin-left: auto;
+	margin-right: auto;
+
+	.newspack-manage-subscriptions-screen__finished {
+		display: block;
+		text-align: center;
+	}
+
+	.newspack-manage-subscriptions-screen__subscription-card {
+		display: flex;
+		justify-content: space-between;
+
+		.newspack-manage-subscriptions-screen__subscription-card__product-actions {
```

Comment:
[Suggestion] Overly nested? [Sass docs](https://sass-lang.com/guide) say `Be aware that overly nested rules will result in over-qualified CSS that could prove hard to maintain and is generally considered bad practice.` and I generally tried to avoid exceeding 2 or 3 levels.